optuna/samplers/_tpe/_truncnorm.py

```diff
@@ -0,0 +1,175 @@
+# This file contains the codes from SciPy project.
+#
+# Copyright (c) 2001-2002 Enthought, Inc. 2003-2022, SciPy Developers.
+# All rights reserved.
+
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+
+# 2. Redistributions in binary form must reproduce the above
+#    copyright notice, this list of conditions and the following
+#    disclaimer in the documentation and/or other materials provided
+#    with the distribution.
```

Comment:
Like the CuPy's documentation, could you add the license page in our documentation?
https://docs.cupy.dev/en/stable/license.html#scipy

Reply:
added a new doc page.